www/docs/server/middlewares.md

```diff
@@ -134,3 +134,87 @@ export const appRouter = router({
   //                                                 ^?
 });
 ```
+
+## Piping(unstable) (extending)
+
+Piping is a powerful feature that allows you to extend middlewares in a typesafe manner.
```

Comment:
```suggestion
## Extending middlewares

:::Info
We have prefixed this as `unstable_` as it's a new API that might change its name in future versions, but the feature is well-tested, is running in production already, and we highly encourage you to use it.
:::

We have a powerful feature called `.pipe()` which allows you to extend middlewares in a typesafe manner.
```

Reply:
Maybe we should add a section in the FAQ what it means to us when we have prefixed something with `unstable_`:

> If you encounter a feature in tRPC that is marked as `unstable_` it means that the API is unstable and might change in minor version bumps, but:
> 
> - We encourage you to use it
> - If it exists in tRPC we're already using it in prod
> - Specifics of the implementation might change in minor changes - its name and options might change
> - If any changes are done to `unstable_features`, they will be included in the release notes
> - Report any suggestion on the API design or issues on https://github.com/trpc/trpc/issues